linalg/blockmatrix.hpp

```diff
@@ -71,6 +71,9 @@ class BlockMatrix : public AbstractSparseMatrix
        treated according to that policy. */
    void EliminateRowCol(int rc, DiagonalPolicy dpolicy = DIAG_ONE);
 
+   void EliminateRowCols(const Array<int> & vdofs, BlockMatrix *Ae,
```

Comment:
There are several new functions in this file that need to be documented.

Reply:
added documentation